sklearn/manifold/t_sne.py

```diff
@@ -629,11 +646,9 @@ class TSNE(BaseEstimator):
     >>> X = np.array([[0, 0, 0], [0, 1, 1], [1, 0, 1], [1, 1, 1]])
     >>> model = TSNE(n_components=2, random_state=0)
     >>> np.set_printoptions(suppress=True)
-    >>> model.fit_transform(X) # doctest: +ELLIPSIS, +NORMALIZE_WHITESPACE
-    array([[ 0.00017619,  0.00004014],
-           [ 0.00010268,  0.00020546],
-           [ 0.00018298, -0.00008335],
-           [ 0.00009501, -0.00001388]])
+    >>> X_embedded = model.fit_transform(X)
+    >>> X_embedded.shape
+    (4, 2)
```

Comment:
Because the values depend on the fact that we fixed the use of the "early exaggeration" parameter in this PR. Furthermore we are likely to change the default values of the optimizers hyperparams to make it match the reference implementation and the actual absolute values the embedding (as long as the trustworthiness score is good). What is the important for the user is that the data has been reduced to 2D and is therefore suitable for visualization as a scatter plot.